packages/front-end/components/Experiment/RiskColumn.tsx

```diff
@@ -26,8 +26,8 @@ export default function RiskColumn({
     metricDefaults
   );
 
-  const winRiskThreshold = row.metric.winRisk || defaultWinRiskThreshold;
-  const loseRiskThreshold = row.metric.loseRisk || defaultLoseRiskThreshold;
+  const winRiskThreshold = row.metric.winRisk ?? defaultWinRiskThreshold;
```

Comment:
`0` was getting overwritten with the default. (Yes, 0% is probably bad practice, but the overwrite behavior seems unintended)